modules/augment/src/averageblur.cpp

```diff
@@ -5,41 +5,41 @@
 #include "precomp.hpp"
 namespace cv { namespace augment {
 
-AverageBlur::AverageBlur(Size _minKernelSize, Size _maxKernelSize)
+AverageBlur::AverageBlur(Size _minKernelSize, Size _maxKernelSize, const std::vector<int>& _borderTypes)
 {
+    if (_borderTypes.size() > 0) borderTypes = _borderTypes;
+
+    else borderTypes = { BORDER_CONSTANT , BORDER_REPLICATE , BORDER_REFLECT};
```

Comment:
Please fix coding style:
```cpp
    if (_borderTypes.size() > 0)
        borderTypes = _borderTypes;
    else
        borderTypes = { BORDER_CONSTANT, BORDER_REPLICATE, BORDER_REFLECT };
```

Reply:
What about the rest of types? 
https://docs.opencv.org/master/d2/de8/group__core__array.html#ga209f2f4869e304c82d07739337eae7c5

Reply:
They are not supported in blur and give exception of non supported borderType

Reply:
```
cv.blur(img, (3, 3), borderType=cv.BORDER_REFLECT_101)
```
works

The rest of them throw exception, I agree.

Reply:
Ok I will add it

Reply:
done